### ISSUE TITLE
[Festive Suggestion]-  Yule, a.k.a. how Monika Celebrates the Holidays

### ISSUE THREAD

**Issue by Retrolovania** [Monika-After-Story/MonikaModDev#6718]
As we all know, Monika does not practice any current religion that happens to exist. However, should you go to see her during Christmastime, she'll still celebrate it with you, even though Christmas is officially regarded as a Christian/Catholic holiday. Why is this?

Many non-Catholics/non-Christians still chose to celebrate Christmas with their families and their friends. They do this by participating in many of the same activities that most people associate with the holiday, like setting up trees, giving each other gifts, et cetera. However, what sets them apart is that none of the typical Catholic/Christian traditions that are typically also recognized during this time (i.e. going to church, singing religious carols) are acknowledged.

This concept of celebrating the holiday season like this is known as Yule. (Yes, this is where the terms "Yuletide" and 'Yule log" get their names.) Yule itself is regarded as one of the oldest celebrations that still exists, dating back to pagan times. While it is currently viewed as a way people celebrate Christmas, the actual event itself technically lasts from the beginning of the winter solstice to New Year's Day.

In short, Monika is _technically_ celebrating the holiday with you in this "Yuletide" manner, and I feel that her acknowledging this as a topic would help to clarify why she chooses to spend the holiday season as such with you(us?).

**Comment by YoraeRasante** [comment]
Monika mentions in a conversation she believes her country is Japan, even if it does not fit much it points to it with everyone else and does not fit outside it either.

In japanese culture, or so I read online, while the name Christmas is _still_ used, it is not about the christian things, but more of a couples' gift sharing day.

It is closer to the usual version of Valentines' Day, since _their_ version of Valentines is also different (girls give chocolate on it, while guys give a gift back the next month on White Day).

So for her it is not a religious thing, but a cultural couples thing.

BUT I agree that a conversation about this would be nice. Her telling the player about Christmas in Japan, and comparing with outside it and with Yule. Maybe lso adding other religions with sacred days around the same days of the year too, all as an outsider since she is not religious of course.

**Comment by ThePotatoGuy** [comment]
[closing until we look into it]
